keps/115-Subgroup-support/README.md

```diff
@@ -0,0 +1,223 @@
+# KEP #115 Subgroup Support
```

Comment:
```suggestion
# KEP-115: Subgroup Support
```

For formatting.

Reply:
Also we still miss a kep.yaml, this is important for later tracking. Sorry for the additional works but it's a standard way for kube.

Reply:
Fixed and added kep.yaml